draft-ietf-sidrops-aspa-verification.xml

```diff
@@ -346,153 +426,232 @@ authorized(AS x, AS y) =  / Else, "Provider+" if the VAP-SPAS entry
       </t>
-    </section>
-
-    <section title="BGP Roles">
       <t>
-        The procedures for local BGP Role announcement in the BGP OPEN message and neighbor role cross-check specified in <xref target="RFC9234"/> are RECOMMENDED.
-        The configured BGP Roles SHOULD be used to automate the use of described above verification procedures helping to distinguish whether upstream or downstream procedures should be applied.
+      <strong>Valid or Unknown</strong>: 
+       When a route is evaluated as Unknown (using ASPA-based AS_PATH verification), it SHOULD be treated at the same preference level as a route evaluated as Valid.
       </t>
+   </section>
+   </section>
 
-      <t>
-        In case of Complex peering relations that can't be segregated in multiple eBGP sessions, an operator may want to achieve an equivalent outcome by configuring policies with corresponding verification procedures on a per-prefix basis.
-      </t>
+  <section title="Deployment Recommendations" anchor="deployment">
+     <t>
+      This section describes practical deployment recommendations.
+     </t>
+   <section title="Implementation at eBGP Ingress" anchor="impl-verif">
+     <t>     
+      The procedures for ASPA-based AS_PATH verification and anomaly mitigation discussed in this document are intended for implementation on eBGP routers on the ingress side.
+      This includes eBGP routers on the boundary of an AS Confederation facing external ASes.
+      However, the procedures are NOT RECOMMENDED for use on internal BGP (iBGP) sessions or eBGP sessions internal to an AS Confederation.
+     </t>
+   </section>
+   <section title="Logging" anchor="logging">
+      <t>
+       For any route with an Invalid AS_PATH, the cause of the Invalid state SHOULD be logged for monitoring and diagnostic purposes.
+       The cause of the Invalid state can be recorded in the form of listing the AS hops which were evaluated by the hop-check function to be &quot;Not Provider+&quot;.
+       The logging router, however, cannot necessarily determine the AS that caused the route leak.
+       For any route with an Unknown AS_PATH, the cause of the Unknown state MAY be logged for monitoring and diagnostic purposes.
+       If logging is done for the Unknown case, the cause can be recorded in the form of listing the AS hops which were evaluated by the hop-check function to be "No Attestation" or "Not Provider+".
+      </t>
+   </section>
+   <section title="BGP Roles" anchor="role">
+     <t>
+      The procedures for local BGP Role announcement in the BGP OPEN message and neighbor role cross-check specified in <xref target="RFC9234"/> are RECOMMENDED.
+      This allows for automatic rejection of a BGP session with a neighbor AS in case there is a mismatch between the neighbor's Role conveyed in BGP OPEN and the one configured locally <xref target="RFC9234"/>.
+      This in turn facilitates more accurate and effective deployment of ASPA.
```

Comment:
How about:
s/This allows for automatic rejection of a BGP session
   with a neighbor AS in case there is a mismatch between the neighbor's
   Role conveyed in BGP OPEN and the one configured locally [RFC9234].
/This allows for automatic rejection of a BGP session
   with a neighbor AS in case there is a mismatch between the neighbor's
   BGP Role conveyed in BGP OPEN and the one configured locally [RFC9234].
   Note that one exception to this is the mutual-transit BGP role which is not 
   considered in [RFC9234] /

Reply:
fine with me